auditorium/src/views/components/auditorium/embed-code.js

```diff
@@ -30,7 +30,7 @@ const EmbedCode = (props) => {
           __html: __('To use Offen with the account <strong>%s</strong> on your website, embed the following script on each page you want to appear in your statistics.', model.account.name)
         }}
       />
-      <div class='w-100 br1 ba b--moon-gray ph2 pv2 bg-light-yellow'>
+      <div class='w-100 br1 ba b--moon-gray ph2 pv2 white bg-silver'>
```

Comment:
This color combo fails the Lighthouse Accessibility Audit. Can we have sth with more contrast?